src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
```
This should not be the case. You just have to make a ColorSupplier object here and use its method.